sympy/core/numbers.py

```diff
@@ -505,6 +513,19 @@ class Float(Number):
     Notes
     =====
 
+    The behaviour when using the null string argument with low-precision can
```

Comment:
This seems confusing to me. Before you try to rewrite, what is the point that you are trying to highlight? That you get a low precision float if you use the ''-arg?